.cafile

```diff
@@ -0,0 +1,8 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
no longer used? (cafile seems to be in `tests/stress/...`  now)

Reply:
I need to clean this up, I've added it 3 places in the file tree over the time...